Mainboard/README.md

```diff
@@ -254,3 +254,28 @@ as "JST SH", and has a 1.0 mm pitch. The same connector is used for the Speakers
 | 2   | FAN_SPEED |
 | 3   | FAN_PWM   |
 | 4   | GND       |
+
+## RTC Battery Interface
+
+NOTE: The mainboard will charge this battery, so a rechargable ML1220 battery must be used.
+CR2032 is not okay because it cannot be charged.
+
+### Onboard RTC Battery
+
+Framework 13 Intel Core 11th through 13th Generation use an onboard coin cell battery as RTC.
+These systems ship with a battery that keeps BIOS time, settings and configuration.
```

Comment:
Might be useful to say ram training data is stored here, so if you don't have a battery the board will take forever to boot, not just the first time but every time.